src/app/starredentries/starredentries.component.ts

```diff
@@ -35,8 +35,10 @@ export class StarredEntriesComponent extends Base implements OnInit {
   // Default to workflows tab
   currentTab = 'workflows';
   selected = new UntypedFormControl();
-  // TO DO: Add 'services' between notebooks and organizations when implemented
-  validTabs = ['workflows', 'tools', 'notebooks', 'organizations'];
+  // TO DO: Add 'services' to validTabs when implemented
```

Comment:
super tiny nitpick: for IDEs to highlight and the like, its one word https://www.osedea.com/en/blog/should-you-put-todos-in-the-source-code

Reply:
I'll fix these while I'm at it (I think most/all are mine LOL)

![image](https://user-images.githubusercontent.com/97123241/231499783-d76930fe-a6f5-4b87-b7b0-adbce98e4e91.png)